design/resources.go

```diff
@@ -355,3 +355,25 @@ var _ = Resource("trackerquery", func() {
 	})
 
 })
+
+var _ = Resource("search", func() {
+	BasePath("/search")
```

Comment:
Technically search should be able to return 'all type' of objects. I think there are 2 different ways to filter the type, by ?type=workitem|user|project|code and as part of the query it self. 

In the UI for the specific use case of linking, the UI would know it can only link workitems(?) so it can auto add that filter. 

But in a more general search bar, the user can filter it via e.g. "type:workitem search term a b"